src/appengine/libs/issue_management/issue_tracker.py

```diff
@@ -74,6 +74,11 @@ def reset(self):
     self._added.clear()
     self._removed.clear()
 
+  def clear(self):
+    """Clear the store, and reset tracking."""
+    self.reset()
```

Comment:
Do we need both reset and clear function. Can we clear backing in all cases. Just the two function names are similar and confusing and i wonder if people will incorrectly use one vs other.

Reply:
Renamed reset to reset_tracking to make the difference clearer. We do need both.